src/test/static/namespaces/import-all.html

```diff
@@ -0,0 +1,4 @@
+<script src="namespace-named.js"></script>
```

Comment:
I test the JS files directly during scanning, but for full analysis its necessary to analyze them all together to test for duplicates across files. Will also be useful to test `memberof` across files.